### DISCUSSION TITLE
InnoDB 설명 중 이해가 안됐던 문장

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#9] [Q&A] [answered]
"innodb에서는 insert update delete 쿼리로 데이터가 변경되는 경우 데이터가 디스크의 데이터 파일로 완전히 저장 될 때까지 기다리지 않아도 된다." 이게 무슨 의미일까요?

**Comment by nahyunLee** [comment, marked as answer]
-  insert, delete, update 작업 -> read.write 필요 
 
MyIsam은 위의 쿼리를 실행하면 클라이언트 스레드가 read,wrtie까지 해야만 끝나지만, 
InnoDB는 클라이언트 스레드가 read하고 백그라운드 스레드가 write하니, 클라이언트 스레드가 write까지 기다리지 않고 다른 일 처리 가능해서 기다리지 않아도 된다는거 아닐까요..?!
